tests/test-war-integration/src/main/resources/logging.properties

```diff
@@ -1,5 +1,5 @@
 com.google.cloud.runtimes.level = FINE
-com.google.cloud.runtimes.handlers = com.google.cloud.logging.LoggingHandler
```

Comment:
From the [documentation](https://docs.oracle.com/javase/8/docs/api/java/util/logging/LogManager.html) `<logger>.handlers` seems to be the right syntax.